core/consensus/babe/impl/block_executor.cpp

```diff
@@ -225,9 +230,13 @@ namespace kagome::consensus {
       }
     }
 
-    logger_->info("Imported block with number: {}, hash: {}",
-                  block.header.number,
-                  block_hash.toHex());
+    auto t_end = std::chrono::high_resolution_clock::now();
+
+    logger_->info(
+        "Imported block with number: {}, hash: {} within {} seconds",
+        block.header.number,
+        block_hash.toHex(),
+        std::chrono::duration<double>(t_end - t_start).count());
```

Comment:
```suggestion
        std::chrono::seconds(t_end - t_start).count());
```

Reply:
`error: no matching conversion for functional-style cast from 'typename common_type<duration<long long, ratio<1, 1000000000> >, duration<long long, ratio<1, 1000000000> > >::type' (aka 'duration<long long, ratio<__static_gcd<ratio<1, 1000000000>::num, ratio<1, 1000000000>::num>::value, __static_lcm<ratio<1, 1000000000>::den, ratio<1, 1000000000>::den>::value> >') to 'std::chrono::seconds' (aka 'duration<long long>')`

Reply:
Then duration_cast.

Reply:
std::chrono::duration_cast<std::chrono::seconds>(t_end - t_start)